pkg/internalapi/conditions.go

```diff
@@ -1,9 +1,14 @@
 package internalapi
 
 const (
-	NodeAvailableConditionFormat   = "Node%sAvailable"
-	NodeProgressingConditionFormat = "Node%sProgressing"
-	NodeDegradedConditionFormat    = "Node%sDegraded"
+	// TODO(rzetelskik): change format to NodeSetup in next API version
+	NodeSetupAvailableConditionFormat   = "Node%sAvailable"
+	NodeSetupProgressingConditionFormat = "Node%sProgressing"
+	NodeSetupDegradedConditionFormat    = "Node%sDegraded"
```

Comment:
> Why Node%s* became conditions for NodeSetup controller?

Because they are being set by it. They are then aggregated together with NodeTune's node conditions in the top-level NodeConfig conditions into the generic Available/Progressing/Degraded conditions.

I understand that what you're proposing is to add one more level of aggregation into `Node<NodeName><ConditionType>` in the NodeConfig controller, and have NodeSetup controller aggregate its conditions into `NodeSetup<NodeName><ConditionType>` instead. I can see the purpose behind it (although it doesn't change much functionally, it's just an additional layer), but naming-wise it would end up being confusing imo, since the existing NodeSetup controller conditions are already in the `<ControllerName>Node<NodeName><ConditionType>` form, and we can't change them?